lib/cursor.js

```diff
@@ -100,6 +101,11 @@ module.exports = class Cursor extends Readable {
     return this.getCursor().then(cursor => cursor.toArray());
   }
 
+  addCursorFlag(flag, value) {
+    this._flags[flag] = value;
```

Comment:
I feel like Mongo should have called it "setCursorFlag" given that it seems that each call will override, rather than add to, the previous value